dex/msgjson/types.go

```diff
@@ -722,8 +722,9 @@ type OrderResult struct {
 // OrderBookSubscription is the payload for a client-originating request to the
 // OrderBookRoute, intializing an order book feed.
 type OrderBookSubscription struct {
-	Base  uint32 `json:"base"`
-	Quote uint32 `json:"quote"`
+	Base           uint32 `json:"base"`
+	Quote          uint32 `json:"quote"`
+	NoSubscription bool   `json:"nosub,omitempty"`
```

Comment:
I didn't notice this in previous review.  It's unused so please remove, but I'm glad we don't have to add fields to the message types.

Reply:
I don't remember adding this. Removing...